src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
```

Comment:
> I can't use `run_command` in this situation, because I'd like for it to display the Git branch here, so that this step can be listed and verified in the subsequent `Then it runs the Git commands` step.

I suggested changing the step that verifies which commands are run to make the branch optional. I don't think the `cd` should display the branch. If you agree, maybe we would want the branch name in the table to say `<none>` (I don't think the branch name being empty would work since we are drying up the branch names).

> it is important on which branch the cd operation happens, since the folder cd'ed out of only exists on a particular branch

This is true for the feature you wrote, but is not guaranteed to always be true.